src/app/components/next-up/next-up.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-next-up',
+  templateUrl: './next-up.component.html',
+  styleUrls: ['./next-up.component.scss']
+})
+export class NextUpComponent implements OnInit {
+  // @Input() artist: string;
```

Comment:
I understand the need to comment these out and use hard coded values, but we will need to undo that before merging in.  Also, the place property does not seem represented in the inputs.  and I think it will be difficult to derive from within the component.

I would also recommend two inputs one for request and one for place.  Request can be the entire request object and you can reference each property in the html like request.song etc...